### DISCUSSION TITLE
Get Web page cookies

### DISCUSSION THREAD

**Discussion by Zer0bytte** [cefsharp/CefSharp#4748] [General]
### Is there an existing issue for this?

- [X] I have searched both open/closed issues, no issue already exists.

### CefSharp Version

121.3.130

### Operating System

Windows 10

### Architecture

x64

### .Net Version

7

### Implementation

WinForms

### Reproduction Steps

N/A

### Expected behavior

N/A

### Actual behavior

![image](https://github.com/cefsharp/CefSharp/assets/85482481/3cda880e-e400-46bc-b24c-94c39cc53849)
How to read these cookies in Cefsharp?
i have called this
  var cookies = await Cef.GetGlobalCookieManager().VisitAllCookiesAsync();
and the return value is different than browser cookies

### Does this problem also occur in the CEF Sample Application

Yes using WPF/OffScreen command line args

**Comment by amaitland** [comment]
In future please use [Discussions](https://github.com/cefsharp/CefSharp/discussions) to ask questions.

> How to read these cookies in Cefsharp?
> i have called this
> var cookies = await Cef.GetGlobalCookieManager().VisitAllCookiesAsync();
> and the return value is different than browser cookies

- When are you calling this code? Please provide a more detailed sample.
- Have you waited for the page to load successfully?
- Are you using multiple `RequestContext` instances?
